src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -493,21 +493,27 @@ internal static string TupleMemberName(int position)
             return "Item" + position;
         }
 
-        // This array needs to be sorted for binary searching
-        private static readonly string[] reservedMemberNames = new[]
-        {
-            "CombineHashCodes",
-            "CompareTo",
-            "Create",
-            "Deconstruct",
-            "Equals",
-            "GetHashCode",
-            "GetHashCodeCore",
-            "Rest",
-            "Size",
-            "ToString",
-            "ToStringEnd"
-        };
+        private static bool IsElementNameForbidden(string name)
+        {
+            switch (name)
+            {
+                case "CombineHashCodes":
+                case "CompareTo":
+                case "Create":
```

Comment:
`ValueTuple` has a public static method `Create`. I thought that if you wrote `var t = (Create: 1, y: 2); M(t.Create);` it would be ambiguous.
Upon testing that, it is un-necessary. I cannot call a static method via an instance. I will remove "Create" from this list.